Src/PerformanceCollector/Shared/Implementation/StandardPerformanceCollector/StandardPerformanceCollector.cs

```diff
@@ -294,9 +294,9 @@ private void RegisterPerformanceCounter(string originalString, string reportAs,
                         usesInstanceNamePlaceholder,
```

Comment:
This was the existing behavior before refactoring. I think its used for retrying once in every 5 mins.(not so sure of this)